terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/gh-runners.ts

```diff
@@ -514,3 +514,13 @@ export async function createRegistrationTokenOrg(
     throw e;
   }
 }
+
+export async function getGitHubRateLimit(installationId: number, metrics: Metrics): Promise<void> {
+  const ghAuth = await createGithubAuth(installationId, 'installation', Config.Instance.ghesUrlApi, metrics);
+  const githubInstallationClient = await createOctoClient(ghAuth, Config.Instance.ghesUrl);
+  const rateLimit = await githubInstallationClient.rateLimit.get();
```

Comment:
Calling the RateLimit API does not count towards the rate limit.

It would be more efficient if we can gather this information from an API call that's already happening rather than doing a separate call to the ratelimit API. However I couldn't find where in the code the best place for that would be so I decided on putting in scale-up.